test/integration/deploy_trigger_test.go

```diff
@@ -588,15 +580,25 @@ func TestTriggers_configChange(t *testing.T) {
 			t.Fatalf("expected watch event type %s, got %s", e, a)
 		}
 
-		if _, err := osClient.DeploymentConfigs(namespace).Update(config); err != nil {
-			return err
-		}
 		return nil
 	})
 	if retryErr != nil {
 		t.Fatal(retryErr)
 	}
 
+	if retryErr := kclient.RetryOnConflict(wait.Backoff{Steps: maxUpdateRetries}, func() error {
+		// submit a new config with an updated environment variable
+		newConfig, err := osClient.DeploymentConfigs(namespace).Generate(config.Name)
```

Comment:
Can you use a simple Get instead of Generate? We have deprecated it anyway and I am removing it in the /instantiate PR.